src/firewall/functions.py

```diff
@@ -458,6 +458,13 @@ def firewalld_is_active():
     return False
 
 
+def removeFile(filename):
+    try:
+        os.remove(filename)
+    except OSError:
```

Comment:
I think not. At least, not for the caller at hand.

The caller is `nft_cmd()`, and that one really does not want to be bothered with this. It certainly shouldn' t just let the exception escape, so, it would need to handle it. But how? Logging a message?

If the caller still needs to handle `OSError` (e.g. because the path is a file, or because the file has wrong permissions), then they should handle `FileNotFoundError` themselves. This function is not for them. This function is for "I don't care about OSError and don't want to write 4 extra lines of code for that" (and that's the only purpose).